### PR TITLE
Cluster config update after kOps update

### PR DESCRIPTION
**Pull request by karanrn**
Added commands to run to update the cluster after kOps update so new features/bug-fixes are applied to the cluster

### CODE REVIEW

File: docs/operations/updates_and_upgrades.md [hunk 1]
```diff
@@ -34,6 +34,12 @@ sudo mv ./kops /usr/local/bin/
 
 You can also rerun [these steps](../contributing/building.md) if previously built from source.
 
+You may want to run below commands to include fixes/features after updating kOps
+```
+kops update cluster <cluster_name> --yes
```
Comment (olemarkus) [inline docs/operations/updates_and_upgrades.md:39]: We use `$NAME` in the rest of the config. Could you change to match?

**Comment by karanrn** [comment]
@olemarkus I have made the change, let me know something needs to added or modified

**Pull request merged**